dipy/reconst/tests/test_dti.py

```diff
@@ -511,37 +511,56 @@ def test_mask():
                                     dtifit.S0_hat[0, 0, 0])
 
 
-def test_nnls_jacobian_fucn():
+def test_nnls_jacobian_func():
     b0 = 1000.
     bval, bvecs = read_bvals_bvecs(*get_fnames('55dir_grad'))
     gtab = grad.gradient_table(bval, bvecs)
     B = bval[1]
 
     # Scale the eigenvalues and tensor by the B value so the units match
-    D = np.array([1., 1., 1., 0., 0., 1., -np.log(b0) * B]) / B
+    D_orig = np.array([1., 1., 1., 0., 0., 1., -np.log(b0) * B]) / B
 
     # Design Matrix
     X = dti.design_matrix(gtab)
 
     # Signals
-    Y = np.exp(np.dot(X, D))
-
-    # Test Jacobian at D
-    args = [X, Y]
-    analytical = dti._nlls_jacobian_func(D, *args)
-    for i in range(len(X)):
-        args = [X[i], Y[i]]
-        approx = opt.approx_fprime(D, dti._nlls_err_func, 1e-8, *args)
-        assert np.allclose(approx, analytical[i])
-
-    # Test Jacobian at zero
-    D = np.zeros_like(D)
-    args = [X, Y]
-    analytical = dti._nlls_jacobian_func(D, *args)
-    for i in range(len(X)):
-        args = [X[i], Y[i]]
-        approx = opt.approx_fprime(D, dti._nlls_err_func, 1e-8, *args)
-        assert np.allclose(approx, analytical[i])
+    Y = np.exp(np.dot(X, D_orig))
+    # NOTE: added error here - maybe remove later
```

Comment:
comment to address and/or remove

Reply:
done, will leave in the addition of error in the test